datalad/cmdline/helpers.py

```diff
@@ -104,13 +104,6 @@ def __call__(self, parser, namespace, values, option_string=None):
             pos_args_str = "*Arguments*"
         helpstr = re.sub(r'optional arguments:', opt_args_str, helpstr)
         helpstr = re.sub(r'positional arguments:', pos_args_str, helpstr)
-        # convert all headings to have the first character uppercase
-        headpat = re.compile(r'^([a-z])(.*):$',  re.MULTILINE)
```

Comment:
I don't mind this PR, and do not remember from top of the head if it would work -- but wouldn't it be possible to add lookbehind for an empty line before this matching one?  smth like
```
In [8]: re.search(r'(?<=^\n)^([a-z])(.*):$',  "with empty\n\nline here:", re.MULTILINE)
Out[8]: <_sre.SRE_Match at 0x7f7c461c7f60>

In [9]: re.search(r'(?<=^\n)^([a-z])(.*):$',  "without empty\nline here:", re.MULTILINE)

In [10]: 
```
but I guess it would also need to account for the beginning of the doc where it should also match